x-pack/plugins/cloud/server/collectors/cloud_usage_collector.ts

```diff
@@ -9,23 +9,34 @@ import { UsageCollectionSetup } from '@kbn/usage-collection-plugin/server';
 
 interface Config {
   isCloudEnabled: boolean;
+  trialEndDate?: string;
+  isElasticStaffOwned?: boolean;
 }
 
 interface CloudUsage {
   isCloudEnabled: boolean;
+  trialEndDate?: string;
+  inTrial?: boolean;
+  isElasticStaffOwned?: boolean;
 }
 
 export function createCloudUsageCollector(usageCollection: UsageCollectionSetup, config: Config) {
-  const { isCloudEnabled } = config;
+  const { isCloudEnabled, trialEndDate, isElasticStaffOwned } = config;
   return usageCollection.makeUsageCollector<CloudUsage>({
     type: 'cloud',
     isReady: () => true,
     schema: {
       isCloudEnabled: { type: 'boolean' },
+      trialEndDate: { type: 'date' },
```

Comment:
I'm a bit confused here, it seems that `CloudUsage` defines the `trialEndDate` as a `string`, whereas this schema defines it as a `Date`. But then looking at the fetch it would seem that we return a `string`. 🤔

Reply:
haha! yeah... `type: 'date'` is a `string` that's supposed to have the format of a `date`.